src/owncloud/apache_config

```diff
@@ -14,3 +14,19 @@ Alias "/apps" "${SNAP_DATA}/owncloud/apps"
 <IfModule dir_module>
     DirectoryIndex index.html index.php
 </IfModule>
+
+
+
+# MPM Event
+<If "${SNAP_ARCH} == 'armhf'"">
```

Comment:
I'm seeing from the email thread that I missed the conversation which happened during previous commits.

I would reverse the order of the if statements then.

Start with `<IfModule event.c>`, then the arch statement and we can also include one for amd64, although from my pov, it's not about the arch, but about the amount of RAM and CPUs. an amd64 VM with 1GB of RAM is not going to do much better (except it has a much better network probably)